unit-tests/heitech.zer0mqXt.core.tests/ConfigurationTestData.cs

```diff
@@ -27,5 +28,11 @@ public IEnumerator<object[]> GetEnumerator()
         }
 
         IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
+
+
+        public static ISocket BuildInprocSocketInstanceForTest(string pipeName)
```

Comment:
Could we make this method `internal`? Since it's named "ForTest" I guess this method shouldn't be externally visible.

Reply:
done